modular_zubbers/modules/code/alt_job_titles.dm

```diff
@@ -0,0 +1,421 @@
+/**
+ * This is the file you should use to add alternate titles for each job, just
+ * follow the way they're done here, it's easy enough and shouldn't take any
+ * time at all to add more or add some for a job that doesn't have any.
+ */
+
+/datum/job/ai
+	alt_titles = list(
+		"AI",
+		"Station Intelligence",
+		"Automated Overseer",
+		"Language Model",
+		"Mind Scan",
+		"Remote Helper Agent",
+	)
+
+/datum/job/assistant
+	alt_titles = list(
+		"Assistant",
+		"Civilian",
+		"Tourist",
+		"Businessman",
+		"Businesswoman",
+		"Trader",
+		"Entertainer",
+		"Freelancer",
+		"Artist",
+		"Off-Duty Staff",
+		"Off-Duty Crew",
+		"Collector",
+		"Maintenance Dweller",
+		"Hobbyist",
+		"Intern",
+		"Visitor",
+		"Gamer",
+	)
+
+/datum/job/atmospheric_technician
+	alt_titles = list(
+		"Atmospheric Technician",
+		"Life Support Technician",
+		"Emergency Fire Technician",
+		"Firefighter",
+		"HVAC Engineer",
+	)
+
+/datum/job/barber
+	alt_titles = list(
+		"Barber",
+		"Salon Manager",
+		"Salon Technician",
+		"Stylist",
+		"Colorist",
+	)
+
+/datum/job/bartender
+	alt_titles = list(
+		"Bartender",
+		"Mixologist",
+		"Barkeeper",
+		"Barista",
+	)
+
+/datum/job/blueshield
+	alt_titles = list(
+		"Blueshield",
+		"Command Bodyguard",
+		"Executive Protection Agent",
+	)
+
+/datum/job/botanist
+	alt_titles = list(
+		"Botanist",
+		"Hydroponicist",
+		"Gardener",
+		"Botanical Researcher",
+		"Herbalist",
+		"Florist",
+		"Farmer",
+	)
+
+/datum/job/bouncer
+	alt_titles = list(
+		"Bouncer",
+		"Service Guard",
+		"Muscle",
+	)
+
+/datum/job/corrections_officer
+	alt_titles = list(
+		"Corrections Officer",
+		"Brig Officer",
+		"Prison Guard",
+	)
+
+/datum/job/captain
+	alt_titles = list(
+		"Captain",
+		"Station Commander",
+		"Commanding Officer",
+		"Site Manager",
+		"Facility Director",
+		"Overseer",
+	)
+
+/datum/job/cargo_technician
+	alt_titles = list(
+		"Warehouse Technician",
+		"Deck Worker",
+		"Mailman",
+		"Union Associate",
+		"Inventory Associate",
+		"Package Handler",
+		"Deliveryman",
+	)
+
+/datum/job/chaplain
+	alt_titles = list(
+		"Chaplain",
+		"Priest",
+		"Preacher",
+		"Reverend",
+		"Oracle",
+		"Pontifex",
+		"Magister",
+		"High Priest",
+		"Imam",
+		"Rabbi",
+		"Monk",
+		"Fanatic",
+		"Zealot",
```

Comment:
Don't think NT would hire people as "Fanatics"

Reply:
You would be surprised at the places that hire fanatics. Though, I do agree that it is a bit extreme to push that envelope, and might cause problems later. I will be keeping Zealot in, though, as that is a Fallout reference.